client/my-sites/email/email-providers-stacked-comparison/billing-interval-toggle/index.tsx

```diff
@@ -0,0 +1,44 @@
+import { useTranslate } from 'i18n-calypso';
+import React, { FunctionComponent } from 'react';
```

Comment:
Nit: we should be able to import the `FunctionComponent` type.
```suggestion
import React from 'react';
import type { FunctionComponent } from 'react';
```